buildlib/pr/io_demo/io-demo.yml

```diff
@@ -34,20 +33,49 @@ parameters:
         duration: 600
         interface: $(roce_iface_cx6)
         tls: "rc_x"
-        test_name: tag_umemh_cx6_rc
-      "server one path compatible on CX4":
-        args: ""
-        initial_delay: 9999 # No interference
+        test_name: am_cx6_rc
+        extra_run_args: ""
+      "tag match on CX6/DC":
```

Comment:
need to remove DC and AM tests from this branch. keep only tag match on CX4/RC

Reply:
but this PR was going to fix tag match on CX6/RC

Reply:
right, keep cx6 but remove AM and DC below